ext/standard/array.c

```diff
@@ -556,6 +934,114 @@ static bucket_compare_func_t php_get_data_compare_func_unstable(zend_long sort_t
 }
 /* }}} */
 
+static zend_always_inline bool php_hash_has_string_keys(HashTable *ht)
```

Comment:
Can't this be const?
```suggestion
static zend_always_inline bool php_hash_has_string_keys(const HashTable *ht)
```